packages/components/input/src/Input.styles.ts

```diff
@@ -35,19 +41,34 @@ export const inputStyles = cva(
         alert: ['border-alert', 'focus:ring-alert'],
         error: ['border-error', 'focus:ring-error'],
       },
+      /**
+       * Sets if there is an addon before the input text.
+       */
       hasLeadingAddon: {
         true: ['rounded-l-none'],
         false: ['rounded-l-lg'],
       },
+      /**
+       * Sets if there is an addon after the input text.
+       */
```

Comment:
I don't think this type of property is exposed to our consumers, right?
same goes for the following `has*` props below

Reply:
Yep, these comments are not exposed into the compiled files and they will only appear in the `docgen.json`